Mathlib/AlgebraicTopology/SimplicialSet/Path.lean

```diff
@@ -85,4 +85,47 @@ lemma Path.ext' {n : ℕ} {f g : Path X (n + 1)}
       rw [← f.arrow_tgt (Fin.last n), ← g.arrow_tgt (Fin.last n), h]
   · exact h j
 
+/-- Maps of simplicial sets induce maps of paths in a simplicial set.-/
+@[simps]
+def mapPath {X Y : SSet.{u}} {n : ℕ} (σ : X ⟶ Y) (f : X.Path n) : Y.Path n where
```

Comment:
Doing as follows would allow the use of dot notation `f.map σ`:
```suggestion
def Path.map {X Y : SSet.{u}} {n : ℕ} (f : X.Path n) (σ : X ⟶ Y) : Y.Path n where
```
This may also apply to other definitions below.